src/intrinsic/mod.rs

```diff
@@ -122,6 +119,7 @@ impl<'a, 'gcc, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'a, 'gcc, 'tcx> {
         let result = PlaceRef::new_sized(llresult, fn_abi.ret.layout);
 
         let simple = get_simple_intrinsic(self, name);
+        #[allow(clippy::suspicious_else_formatting)]
```

Comment:
Please add a `FIXME` with the link to this new issue that I opened for this bug: https://github.com/rust-lang/rust-clippy/issues/12497

Reply:
Done in tempdragon@3b01fba0f5ecfd7e929fcf1948945ae75479c38d